mbed_lstools/platform_database.py

```diff
@@ -47,7 +47,13 @@
     u'0003': u'LPC2368',
     u'0004': u'LPC2368',
     u'0005': u'LPC2368',
-    u'0006': u'LPC2368',
+    u'0006': {
+        u'default': 'daplink',
```

Comment:
when this default is needed?

Reply:
If someone invokes the `platform_db.get` api without specifying a `device_type`, then we need to know which platform mapping we need to use. I'm open to restructuring that bit if needed, it was just a quick pass at it.